helper/consts/replication.go

```diff
@@ -119,21 +119,12 @@ func (r *ReplicationState) AddState(flag ReplicationState)     { *r |= flag }
 func (r *ReplicationState) ClearState(flag ReplicationState)   { *r &= ^flag }
 func (r *ReplicationState) ToggleState(flag ReplicationState)  { *r ^= flag }
 
-func (r *ReplicationState) IsLeader() bool {
-	// Is Vault clustered and this instance is the leader?
-	if r.HasState(ReplicationPerformancePrimary) {
-		return true
-	}
-	if r.HasState(ReplicationDRPrimary) {
-		return true
-	}
-	// Is replication disabled so Vault is standalone?
-	if r.HasState(ReplicationPerformanceDisabled) {
+func (r *ReplicationState) IsFollower() bool {
+	if r.HasState(ReplicationPerformanceSecondary) {
 		return true
 	}
-	if r.HasState(ReplicationDRDisabled) {
+	if r.HasState(ReplicationPerformanceStandby) {
```

Comment:
For reasons that are not clear to me, in backends via system this will work but in the vault package the replication state variable will not contain perf standby. So calling this function will sometimes work the way you expect and sometimes not

Reply:
Rather I know why it is this way, dynamic system view adds it, but I don't know why it isn't in replication state in core to begin with.